test/helpers/foundry/foundryup.test.ts

```diff
@@ -0,0 +1,219 @@
+import { join } from 'path';
+import { homedir } from 'os';
+import fs from 'fs/promises';
+import { readFileSync } from 'fs';
+import { parse as parseYaml } from 'yaml';
+// Import test helpers
```

Comment:
we could use functions from froundryup file, if the logic is split into smaller functions